tests/vnc_test.go

```diff
@@ -61,15 +61,12 @@ import (
 var _ = Describe("[rfe_id:127][crit:medium][arm64][vendor:cnv-qe@redhat.com][level:component][sig-compute]VNC", decorators.SigCompute, func() {
 
 	var err error
-	var virtClient kubecli.KubevirtClient
```

Comment:
Wish we could also remove the `err`

Reply:
Oh dear. `err` is mentioned in a maze of nested functions, some of them masking the original one with `:=`. With your permission I would consider it in a follow-up https://github.com/kubevirt/kubevirt/pull/11340 .